<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>必要信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 126);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="set_num">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 255, 255);</string>
    </property>
    <property name="title">
     <string>请输入无人船数量：</string>
    </property>
    <widget class="QLineEdit" name="ask_num">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 246, 247);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>291</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 255, 255);</string>
    </property>
    <property name="title">
     <string>请输入本机ip地址：</string>
    </property>
    <widget class="QLineEdit" name="ask_ip">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 246, 247);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
